pkg/api/types.go

```diff
@@ -138,6 +138,17 @@ type ObjectMeta struct {
 	// objects.  Annotation keys have the same formatting restrictions as Label keys. See the
 	// comments on Labels for details.
 	Annotations map[string]string `json:"annotations,omitempty"`
+
+	// List of objects depended by this object. If ALL objects in the list have
+	// been deleted, this object will be garbage collected.
+	OwnerReferences []ObjectReference `json:"ownerReferences,omitempty"`
+
+	// Must be empty before the object is deleted from the registry. The map key
+	// is an ad hoc, explanatory name (e.g., "orphaning finalizer"), the value
```

Comment:
I'll switch the meaning of the key and the value, because a map key needs to be unique, while an "explanatory name" doesn't have to be.